Sources/App/Controllers/UserController.swift

```diff
@@ -12,6 +12,10 @@ import AuthProvider
 import JWT
 
 final class UserController {
+    var droplet: Droplet
```

Comment:
Since this is being initialized right away, there isn't a need to have it as a `var` You can make it a constant